### PR TITLE
Add management page service layer interface with mock data

### PR DESCRIPTION
**Pull request by wjrjerome**
Empty class with types for handling the management page

### CODE REVIEW

File: frontend/src/services/grandmaster-manager/index.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import Web3 from "web3";
+import { ManagerError } from "./errors";
```
Comment (trust2065) [inline frontend/src/services/grandmaster-manager/index.ts:2]: I usually avoid using relative route

File: frontend/src/services/grandmaster-manager/index.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  
+  /**
+   * * This method will detect XDC-Pay and verify if customer has alraedy loggin.
+   * @returns Account details will be returned if all good, otherwise, relevant error message and type will be returned such as WALLET_NOT_LOGIN
+   */
+  async login(): Promise<AccountDetails | ManagerError> {
+    return {
+      accountAddress: "xdc888c073313b36cf03cf1f739f39443551ff12bbe",
+      balance: "123",
+      chainId: 551
```
Comment (trust2065) [inline frontend/src/services/grandmaster-manager/index.ts:26]: From the design we need redeemable balance, network id, denom and RPC

Comment (trust2065) [reply frontend/src/services/grandmaster-manager/index.ts:26]: Also where is `chainId` being used for

**Review by trust2065 — CHANGES_REQUESTED** [review]

File: frontend/src/services/grandmaster-manager/index.ts [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  
+  /**
+   * A method to return subnet candidates and its status.
+   * @returns The address and its current status and stake.
+   * 'MASTERNODE' means it's one of the mining masternode
+   * 'PROPOSED' means it just been proposed, but waiting for have enough vote in order to be the masternode.
+   * 'SLASHED' means it's been taken out from the masternode list
+   */
+  async getCandidates(): Promise<{[address: string]: { capacity: number, status: 'MASTERNODE' | 'PROPOSED' | 'SLASHED'}}> {
+    return {
```
Comment (trust2065) [inline frontend/src/services/grandmaster-manager/index.ts:76]: Could this be an array with the shape 
```
[{
  address: xxx,
  delegation: xxx,
  rank: xxx,
  status: xxx
}]
```

I don't know what `capacity` is used for as well by looking at the design

File: frontend/src/services/grandmaster-manager/index.ts (hunk 2 above)
Comment (wjrjerome) [reply frontend/src/services/grandmaster-manager/index.ts:26]: network id is the chainId. I can help change that term here within this layer
denom is something i have missed. i will add it here but i haven't figure out where/how i can get this value

File: frontend/src/services/grandmaster-manager/index.ts (hunk 3 above)
Comment (wjrjerome) [reply frontend/src/services/grandmaster-manager/index.ts:76]: can do that. the current format is basically what's been returned from the API.

Comment (wjrjerome) [reply frontend/src/services/grandmaster-manager/index.ts:76]: capacity is the same as delegation, this is the term been used in the node side (BE side)

**Pull request merged**